.github/workflows/pr-review.yml

```diff
@@ -0,0 +1,19 @@
+name: PR review 
+
+on: [pull_request]
+
+jobs:
+  # Danger is a tool that can help during code reviews. Automate common 
+  # code review tasks. https://danger.systems/js/
+  # TL;DR - it runs script `dangerfile.js` on each pull request. 
+  danger:
+    runs-on: ubuntu-latest
+    name: Danger 
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run danger 
+        run: npx danger@11 ci --dangerfile dangerfile.js
```

Comment:
I added a [new tool](https://danger.systems/js/) to our workflow in this PR. I don't like adding too many different tools to a project to add complexity and require more learning by team members to learn each tool. Luckily, this is a simple tool that simply runs a script on each PR to help us automate parts of the pull request review. 

Right now, the script has only 1 check in it but we can do more in the future.